cupy/cuda/memory.pyx

```diff
@@ -1573,14 +1570,17 @@ cdef class MemoryAsyncPool:
     cdef:
         # A list of cudaMemPool_t to each device's mempool
         readonly list _pools
+        bint _memoryAsyncHasStat
+
+
```

Comment:
Just two nits:
1. We can just name it as `memoryAsyncHasStat` since it's not exposed to Python; you could also mark it `readonly` if you like.
2. The two extra lines are not needed.